clayui.com/content/docs/get-started/using-clay-in-jsps.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Using Clay in JSPs'
+order: 6
+---
+
+<div class="nav-toc-absolute">
+<div class="nav-toc">
+
+-   [Setting up your module](#setting-up-your-module)
+-   [Using Clay taglib in your JSPs](#using-clay-taglib-in-your-jsps)
+-   [Clay components available in JSPs](#clay-components-available-in-jsps)
```

Comment:
If we scrap the setting up module and gradle portion, which I think we should so that we don't have to concern ourselves too much with the dxp build specifics, I think we could structure the page like...

```suggestion
-   [Using Taglibs](#using-taglibs)
-   [Clay taglibs available](#clay-taglibs-available)
```